tests/main/refresh-app-awareness-notify/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Ensure that refresh app-awareness notifications work
+
+details: |
+    Test that notification for continued auto-refreshes are delivered
+
+# Ubuntu 14.04's special version of systemd doesn't have StartTransientUnit API.
+systems: [-ubuntu-14.04-*]
+
+restore: |
+    snap unset system refresh.timer
+
+execute: |
+    # setup test snap and make sure an app keeps it busy
+    
+    snap install test-snapd-sh
+    test-snapd-sh.sh -c 'exec sleep infinity' &
+    PID="$!"
+    # Note that this relies on having different revisions between
+    # test-snapd-sh stable and edge (but other tests require this too)
+    snap switch --edge test-snapd-sh
+
+    # trigger auto-refresh
+    # XXX: do we need this?
```

Comment:
I don't know why but sometimes there's a pre-existing hold when the spread test starts